app/models/miq_server/worker_management/monitor.rb

```diff
@@ -156,7 +156,7 @@ def check_not_responding
   end
 
   def monitor_reason_not_responding?(w)
-    [MiqServer::NOT_RESPONDING, MiqServer::MEMORY_EXCEEDED].include?(worker_get_monitor_reason(w.pid)) || w.stopping_for_too_long?
+    [MiqServer::WorkerManagement::NOT_RESPONDING, MiqServer::WorkerManagement::MEMORY_EXCEEDED].include?(worker_get_monitor_reason(w.pid)) || w.stopping_for_too_long?
```

Comment:
think both of these are only used in monitor.
can get rid of some namespacing if you put i there